cmrtlib/linux/CMakeLists.txt

```diff
@@ -55,8 +55,17 @@ include_directories(${CMAKE_CURRENT_LIST_DIR}/../agnostic/share)
 include_directories(${CMAKE_CURRENT_LIST_DIR}/../agnostic/hardware)
 include_directories(${CMAKE_CURRENT_LIST_DIR}/../linux/share)
 include_directories(${CMAKE_CURRENT_LIST_DIR}/../linux/hardware)
+
 if(NOT "${LIBVA_INSTALL_PATH}" STREQUAL "")
     include_directories(BEFORE ${LIBVA_INSTALL_PATH})
+else()
+    include(FindPkgConfig)
```

Comment:
what's the prerequisite to make this FindPkgConfig work properly?
I just build libva below way in Ubuntu 16.04 -
$ ./autogen.sh --prefix=/usr --libdir=/usr/lib/x86_64-linux-gnu
$ make
$ sudo make install
Then below LIBVA_INCLUDE_DIRS and LIBVA_X11_INCLUDE_DIRS are empty

Another strange thing is if I change the judgement as libva>=2.0.0 (even 3.0.0), LIBVA_FOUND will still be true. Same for LIBVA_X11.

I checked by pkg-config -
$ pkg-config --modversion libva
1.0.0
And /usr/lib/x86_64-linux-gnu/pkgconfig/libva.pc also told me -
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
Name: libva
Description: Userspace Video Acceleration (VA) core interface
Version: 1.0.0
<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<

What I missed? Thanks

Reply:
>what's the prerequisite to make this FindPkgConfig work properly?

The prerequisite is that the `libva.pc` file must be in the pkg-config search path.  In your case, it is (i.e. `/usr/lib/x86_64-linux-gnu/pkgconfig`).  If the .pc file is not in the default search path, then user can set `PKG_CONFIG_PATH` environment variable to give pkg-config additional paths to search.

> Then below LIBVA_INCLUDE_DIRS and LIBVA_X11_INCLUDE_DIRS are empty

`LIBVA_INCLUDE_DIRS` and `LIBVA_X11_INCLUDE_DIRS` will be empty if its directories are already included in the default cmake include paths.

> Another strange thing is if I change the judgement as libva>=2.0.0 (even 3.0.0), LIBVA_FOUND will still be true. Same for LIBVA_X11.

I was able to reproduce your findings for `libva>=2.0.0` but not for `libva-x11>=2.0.0`.  Turns out that the previous call to `pkg_check_modules(LIBVA REQUIRED libva>=1.0.0)` in `media_driver/cmake/linux/media_include_paths_linux.cmake` overshadows the second call... I will fix this.